src/test/scala/gitbucket/core/api/ApiSpecModels.scala

```diff
@@ -42,228 +48,609 @@ object ApiSpecModels {
+    account.copy(
+      isAdmin = true,
+      isGroupAccount = true,
+      description = Some("Admin group")
+    )
+  )
+
+  val apiPlugin = ApiPlugin(
+    plugin = PluginInfo(
+      pluginId = "gist",
+      pluginName = "Gist Plugin",
+      pluginVersion = "4.16.0",
+      gitbucketVersion = Some("4.30.1"),
+      description = "Provides Gist feature on GitBucket.",
+      pluginClass = null,
+      pluginJar = new java.io.File("gitbucket-gist-plugin-gitbucket_4.30.0-SNAPSHOT-4.17.0.jar"),
+      classLoader = null
+    )
+  )
+
+  val apiPusher = ApiPusher(account)
+
+  val apiRef = ApiRef(
+    ref = "refs/heads/featureA",
+    `object` = ApiObject(sha1)
+  )
+
+  // JSON String for APIs
+
+  val jsonUser = """{
+       |"login":"octocat",
+       |"email":"octocat@example.com",
+       |"type":"User",
+       |"site_admin":false,
+       |"created_at":"2011-04-14T16:00:49Z",
+       |"id":0,
+       |"url":"http://gitbucket.exmple.com/api/v3/users/octocat",
+       |"html_url":"http://gitbucket.exmple.com/octocat",
+       |"avatar_url":"http://gitbucket.exmple.com/octocat/_avatar"
+       |}""".stripMargin
+
+  val jsonRepository = s"""{
+       |"name":"Hello-World",
+       |"full_name":"octocat/Hello-World",
+       |"description":"This your first repo!",
+       |"watchers":0,
+       |"forks":1,
+       |"private":false,
+       |"default_branch":"master",
+       |"owner":$jsonUser,
+       |"id":0,
+       |"forks_count":1,
+       |"watchers_count":0,
+       |"url":"http://gitbucket.exmple.com/api/v3/repos/octocat/Hello-World",
+       |"http_url":"http://gitbucket.exmple.com/git/octocat/Hello-World.git",
+       |"clone_url":"http://gitbucket.exmple.com/git/octocat/Hello-World.git",
+       |"html_url":"http://gitbucket.exmple.com/octocat/Hello-World"
+       |}""".stripMargin
+
+  val jsonLabel =
+    """{"name":"bug","color":"f29513","url":"http://gitbucket.exmple.com/api/v3/repos/octocat/Hello-World/labels/bug"}"""
+
+  val jsonIssue = s"""{
+       |"number":1347,
+       |"title":"Found a bug",
+       |"user":$jsonUser,
+       |"labels":[$jsonLabel],
+       |"state":"open",
+       |"created_at":"2011-04-14T16:00:49Z",
+       |"updated_at":"2011-04-14T16:00:49Z",
+       |"body":"I'm having a problem with this.",
+       |"id":0,
+       |"comments_url":"http://gitbucket.exmple.com/api/v3/repos/octocat/Hello-World/issues/1347/comments",
+       |"html_url":"http://gitbucket.exmple.com/octocat/Hello-World/issues/1347"
+       |}""".stripMargin
+
+  // TODO comments_url is correct?
+  val jsonIssuePR = s"""{
+       |"number":1347,
+       |"title":"new-feature",
+       |"user":$jsonUser,
+       |"labels":[$jsonLabel],
+       |"state":"closed",
+       |"created_at":"2011-04-14T16:00:49Z",
+       |"updated_at":"2011-04-14T16:00:49Z",
+       |"body":"Please pull these awesome changes",
+       |"id":0,
+       |"comments_url":"http://gitbucket.exmple.com/api/v3/repos/octocat/Hello-World/issues/1347/comments",
```

Comment:
Is this `comments_url` correct?

Reply:
Probably this is correct. I suppose that since the comments API for pull request seems to be not implemented yet, this property points API for the issues instead.

Reply:
ok